### DISCUSSION TITLE
I can only equip item number 15 on only 1 character, but there are 2 of them.

### DISCUSSION THREAD

**Discussion by sul3z3ro** [Lurkars/gloomhavensecretariat#291] [Q&A] [answered]
My Banner Spear currently equips Reinforced Shield (item number 15). My friend decided to craft one more for himself which is the 2nd Reinforced Shield. However, the App doesn't let me craft the 2nd Reinforced Shield in my friend character sheet, despite the hard copy of Frosthaven which has 2 of Reinforced Shield.

Is this a bug that limit Reinforced Shield to be only available for 1 character per party?

**Comment by Lurkars** [comment]
Definetly a bug, will fix

**Comment by Lurkars** [comment, marked as answer]
Should be fixed in [v0.65.1](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.65.1). I found some more wrong item counts, so thanks for pointing that out. To avoid future issues with wrong data (and to maybe support some house rules), I add functionality to force adding an item by double click (long press on mobile) on plus symbol, ignoring any constrains like count or already owned etc.
